src/map/battle.cpp

```diff
@@ -5298,6 +5384,10 @@ static struct Damage initialize_weapon_data(struct block_list *src, struct block
 				//Fall through
 			case KN_SPEARSTAB:
 			case KN_BOWLINGBASH:
```

Comment:
If user equips two-handed sword, number of hit is increased up to 4 hits depends on number of target.
the damage hits increases on the target monster or target and nearby monsters?